engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -64,6 +64,8 @@
 
 	private static final Log log = LoggerFactory.make();
 
+	private static Boolean missingElDependencies;
```

Comment:
Should this not better be volatile to safely publish the write to other threads?